### PR TITLE
support Symfony 4

### PR DESCRIPTION
**Pull request by othillo**
fixes https://github.com/qandidate-labs/qandidate-toggle-bundle/pull/52

- [ ] update CHANGELOG
- [ ] tag 1.0

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,17 +2,27 @@ language: php
 sudo: false
 
 php:
-  - 5.6
```
Comment (alcaeus) [inline .travis.yml]: These changes should be reflected in `composer.json`. As it is, the bundle is installable on 5.5.9 (due to the implicit requirement through `qandidate/toggle`) but would most likely fail.

File: .travis.yml [hunk 2]
```diff
@@ -2,17 +2,27 @@ language: php
...
+  - SYMFONY_VERSION=2.7.*
+  - SYMFONY_VERSION=2.8.*
+  - SYMFONY_VERSION=3.3.*
+  - SYMFONY_VERSION=3.4.*
+  - SYMFONY_VERSION=4.0.*
 
 matrix:
   include:
-    - php: 5.6.29
-      env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
```
Comment (alcaeus) [inline .travis.yml]: This should probably be kept, just not for 5.6.29 but rather 7.1.

File: .travis.yml (hunk 1 above)
Comment (othillo) [reply .travis.yml]: good point! added php requirement to composer.json

Comment (alcaeus) [reply .travis.yml]: Thanks! Stumbled on this by accident while looking for a good feature switch library, looking forward to using this! ❤️

**Pull request merged**